src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure {
```

Comment:
```suggestion
public abstract class Figure implements Drawable {
```
так тобі не потрібно буде це писати в кожному класі насліднику